### PR TITLE
EcalClusterLazyTools: new n5x5() call analogous to e5x5()

### PR DESCRIPTION
**Pull request by konush**

### CODE REVIEW

**Comment by degano** [comment]
@cmsbuild please test

**Comment by degano** [comment]
@cmsbuild please test

**Comment by davidlange6** [comment]
please test

**Comment by degano** [comment]
@cmsbuild please test

File: RecoEcal/EgammaCoreTools/interface/EcalClusterTools.h [hunk 1]
```diff
@@ -177,6 +178,7 @@ class EcalClusterToolsT {
                 // get the energy deposited in a matrix centered in the maximum energy crystal = (0,0)
                 // the size is specified by ixMin, ixMax, iyMin, iyMax in unit of crystals
                 static float matrixEnergy( const reco::BasicCluster &cluster, const EcalRecHitCollection *recHits, const CaloTopology* topology, DetId id, int ixMin, int ixMax, int iyMin, int iyMax );
+                static float matrixSize( const reco::BasicCluster &cluster, const EcalRecHitCollection *recHits, const CaloTopology* topology, DetId id, int ixMin, int ixMax, int iyMin, int iyMax );
```
Comment (slava77) [inline RecoEcal/EgammaCoreTools/interface/EcalClusterTools.h]: why is this a float?

Comment (slava77) [inline RecoEcal/EgammaCoreTools/interface/EcalClusterTools.h]: please change to int, to avoid int -> float -> int conversion cycle

**Comment by konush** [comment]
Corrected.

**Comment by konush** [comment]
@cmsbuild please test

**Comment by degano** [comment]
@cmsbuild please test

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/7388	13ac9b7
the code looks good now; jenkins compiled it happily.

**Pull request merged**